stbt_rig.py

```diff
@@ -400,6 +446,14 @@ def iter_portal_auth_tokens(portal_url, portal_auth_file, mode):
                 "variable bamboo.STBT_AUTH_PASSWORD")
         return
 
+    if mode == "gitlab":
+        token = os.environ.get("CI_BUILD_TOKEN")
```

Comment:
GitLab's CI_BUILD_TOKEN (now called CI_JOB_TOKEN) is to access the GitLab API during the job. It isn't to access the Stb-tester Portal.